rest/client/rest.go

```diff
@@ -711,6 +712,39 @@ func (c *communicatorImpl) DeleteCommitQueueItem(ctx context.Context, projectID,
 	return nil
 }
 
+func (c *communicatorImpl) EnqueueItem(ctx context.Context, id string) (int, error) {
+	info := requestInfo{
```

Comment:
implementing the REST route is a later ticket [(EVG-5933)](https://jira.mongodb.org/browse/EVG-5933).
This won't actually work until it's implemented.